classes/PodsData.php

```diff
@@ -1881,12 +1864,7 @@ public static function table_truncate( $table ) {
 	 */
 	public static function table_drop( $table ) {
 
-		/**
-		 * @var $wpdb wpdb
-		 */
-		global $wpdb;
-
-		$sql = "DROP TABLE `{$wpdb->prefix}" . self::$prefix . "{$table}`";
+		$sql = "DROP TABLE `" . self::get_pods_prefix() . "{$table}`";
```

Comment:
:no_entry_sign: String "DROP TABLE `" does not require double quotes; use single quotes instead

<sub>hash: 0fe237e7ecb892c22ed19acb0aa40df6</sub>